frontend/integration-tests/tests/networking.scenario.ts

```diff
@@ -0,0 +1,45 @@
+import { appHost, testName, checkLogs, checkErrors } from '../protractor.conf';
+import { execSync } from 'child_process';
+import * as networkingView from '../views/networking.view';
+import { $$} from 'protractor';
+import * as secretsView from '../views/secrets.view';
+import * as crudView from '../views/crud.view';
+
+
+describe('Interacting with route creation forms', () => {
+  beforeAll(async() => {
+    execSync(`kubectl create -f ./integration-tests/data/caddy-docker.json -n ${testName}`);
+    execSync(`kubectl create -f ./integration-tests/data/service-unsecure.json -n ${testName}`);
+  });
+
+  beforeEach(async()=> networkingView.visitRoutesPage(appHost, testName));
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  afterAll(async() => {
+    execSync(`kubectl delete pod caddy-docker -n ${testName}`);
+    execSync(`kubectl delete service service-unsecure -n ${testName}`);
+  });
+
+  const unsecureServiceName = 'service-unsecure';
+
+  it('Unsecured route should prefix with http ', async() => {
+    const unsecureRouteName = 'my-unsecure-route';
+    await networkingView.createUnsecureRoute(unsecureRouteName, unsecureServiceName);
+    await networkingView.visitRoutesDetailsPage(appHost, testName, unsecureRouteName);
+    await crudView.isLoaded();
+    const hostSearchLabel = 'LOCATION';
```

Comment:
This will likely break after #1465. But it would be much better to add a `data-test-` attribute to find the value.